stock_picking_invoicing_incoterm/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+================================
+Stock Picking Invoicing Incoterm
+================================
```

Comment:
Can you split the `README.rst` file to different markdown files in the `readme/` subdirectory of the module?

This will enable building it with our tools, and keeping it up to date with the documentation standards of the OCA.

Reply:
nice catch! I didn't notice `readme/` was missing